src/components/AppComponent.tsx

```diff
@@ -216,11 +214,46 @@ const AppComponent: FC = () => {
               </div>
             ) : (
               <SplitPane
-                className={css({
-                  position: 'relative',
-                  '& .Pane': {
-                    transition: isSplitting ? 'width 0.2s ease' : undefined,
+                paneClassName={css({ transition: isSplitting ? 'width 0.2s ease' : undefined, userSelect: 'none' })}
+                resizerClassName={css({
+                  background: '#fff',
+                  opacity: 0.2,
+                  zIndex: 'resizer',
+                  boxSizing: 'border-box',
+                  backgroundClip: 'padding-box',
+                  userSelect: 'none',
+                  '&:hover': {
+                    transition: 'all 0.2s ease-out',
+                  },
+                  '&.horizontal': {
+                    height: '11px',
+                    margin: '-5px 0',
+                    borderTop: '5px solid rgba(255, 255, 255, 0)',
+                    borderBottom: '5px solid rgba(255, 255, 255, 0)',
+                    cursor: 'row-resize',
+                    width: '100%',
+                  },
+                  '&.horizontal:hover': {
+                    borderTop: '5px solid rgba(0, 0, 0, 0.5)',
+                    borderBottom: '5px solid rgba(0, 0, 0, 0.5)',
                   },
+                  '&.vertical': {
+                    width: '11px',
+                    margin: '0 -5px',
+                    borderLeft: '5px solid rgba(255, 255, 255, 0)',
+                    borderRight: '5px solid rgba(255, 255, 255, 0)',
+                    cursor: 'col-resize',
+                  },
+                  '&.vertical:hover': {
+                    borderLeft: '5px solid rgba(255, 255, 255, 0.5)',
+                    borderRight: '5px solid rgba(255, 255, 255, 0.5)',
+                  },
+                  '&.disabled': { cursor: 'not-allowed' },
+                  '&.disabled:hover': { borderColor: 'transparent' },
+                })}
+                className={css({
+                  position: 'relative', // not applied due to `position: absolute` in style prop of react-split-pane
```

Comment:
Is this referring to https://github.com/tomkp/react-split-pane/blob/master/src/SplitPane.js#L269?

If the relative position is not being applied, should we simply remove it? Or is there a particular reason that we want to set the position to relative, and we need to look into a fix for this?

Reply:
Good questions. We're actually not using the SplitPane at all right now. The implementation proved more difficult than expected due to how tightly coupled the selectors are to `state.cursor`. 

Should probably be removed completely in a separate PR until a proper implementation can be built. We can always go back in the commit history to see how the SplitPane component was originally configured.